release/generate_manifest.sh

```diff
@@ -196,9 +204,9 @@ function get_istio_green_sha() {
   set +e
   # GITHUB_KEYFILE has the github tokens
   local GREEN_SHA
-  GREEN_SHA=$(githubctl --token_file="${GITHUB_KEYFILE}" --repo=istio \
-                        --op=getLatestGreenSHA           --base_branch="${CUR_BRANCH}" \
-                        --logtostderr)
+  GREEN_SHA=$($githubctl --token_file="${GITHUB_KEYFILE}" --repo=istio \
```

Comment:
extra $(?

Reply:
no githubctl is a variable containing the path of the binary, it is not in path